README.md

```diff
@@ -46,6 +46,48 @@ sure to install the new TALs using
 routinator init --force
 ```
 
+## Quick Start with Debian and Ubuntu Packages
+
+### Disclaimer
+> These packages are provided on a best effort basis as a convenience for our community until such time as equivalent official operating system repository provided packages become available.
+
+Assuming you have a machine running a recent Debian or Ubuntu distribution, you
+can install Routinator from our [software package
+repository](https://packages.nlnetlabs.nl). To use this repository, add the line
+below that corresponds to your operating system to  your `/etc/apt/sources.list`
+or `/etc/apt/sources.list.d/`
+
+```bash
+deb [arch=amd64] https://packages.nlnetlabs.nl/linux/debian/ stretch main
+deb [arch=amd64] https://packages.nlnetlabs.nl/linux/debian/ buster main
+deb [arch=amd64] https://packages.nlnetlabs.nl/linux/ubuntu/ xenial main
+deb [arch=amd64] https://packages.nlnetlabs.nl/linux/ubuntu/ bionic main
+deb [arch=amd64] https://packages.nlnetlabs.nl/linux/ubuntu/ focal main
+```
+Then run the following commands.
+
+```bash
+sudo apt update && apt-get install -y gnupg2
+wget -qO- https://packages.nlnetlabs.nl/aptkey.asc | sudo apt-key add -
+sudo apt update
+```
+
+You can then install, initialise, enable and start Routinator by running these
+commands. Note that `routinator-init` is slightly different than the command
+used in Cargo.
+
+```bash
+sudo apt install routinator
+sudo routinator-init
+# Follow instructions provided
+systemctl enable --now routinator
+```
+
+routinator-init
```

Comment:
Sorry, I don't get it, what is this? `routinator-init` is arleady mentioned above.

Reply:
Oops